### ISSUE TITLE
Per player render distance

### ISSUE THREAD

**Issue by Naharie** [Lauriethefish/BetterPortals#17]
tl;dr: Please add a section to the config file allowing me to override the base render distance for specific players based on player name.

Not exactly an issue, but would be a nice to have. My friends are mostly playing on laptops and such, so anything past 13 is pushing it. I on the other hand have a much nicer device and can handle render distances of 30+ with only rare fps drops, and even then those are barely noticeable. In the current state of the plugin the only reasonable thing to do is set the render distance lower so everyone can enjoy it. However if I could set an override for each player, I could customize the distances to match each devices connection speed and such for a even better experience.

Regardless, thank you very much for making this at all! I'm already enjoying it a lot, and hope you continue to maintain it for future versions.

**Comment by Lauriethefish** [comment]
This is not an easy feature to add - portals share a block map between multiple players for the purposes of performance, so separating it would add quite a lot of complexity and also hurt server-side performance, which is usually my priority. (essentially, the plugin builds a map of blocks around the portal that are not fully obscured by other solid blocks, then update this every 20 ticks. Then, a subthread can manage the rendering for each player)

I do understand your situation though - this must be annoying.
It could come in a future update, but if it does, it will be good while - don't hold your breath.

Might come sooner if I find a significantly easier implementation that doesn't hurt performance.

**Comment by Naharie** [comment]
That makes sense. I guess I'll just try to tweak the value to a range that works for everyone. That and hope you get hit by a stroke of pure genius :)
